app/code/core/Mage/Catalog/Model/Resource/Product/Action.php

```diff
@@ -74,6 +74,10 @@ public function updateAttributes($entityIds, $attrData, $storeId)
                 $this->_processAttributeValues();
             }
             $this->_getWriteAdapter()->commit();
+
+            $this->getWriteConnection()->update($this->getTable('catalog/product'), [
+                'updated_at' => Varien_Date::now()
+            ], $this->getWriteConnection()->quoteInto('entity_id IN (?)', $entityIds));
```

Comment:
is it possible that the IN clause becomes too long (in case of many many products) and breaks?

Reply:
I think we should split the query in blocks of 1000 records like shown a few lines above :-)